lms/templates/instructor/instructor_dashboard_2/student_admin.html

```diff
@@ -85,6 +85,49 @@
   <hr>
 </div>
 
+% if course.entrance_exam_enabled:
+<div class="entrance-exam-grade-container action-type-container">
+  <h2>${_("Entrance Exam grade adjustment")}</h2>
+  <div class="request-response-error"></div>
```

Comment:
It would be good if we could reuse the standard error pattern that has been used on recent instructor dashboard projects. If this was a Backbone view it could reuse the NotificationView.